Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -191,4 +196,31 @@ namespace AudioCommon
 		g_sound_stream->GetMixer()->StopLogDSPAudio();
 		s_audio_dump_start = false;
 	}
+
+	void IncreaseVolume(unsigned short offset)
+	{
+		SConfig::GetInstance().m_IsMuted = false;
+		int& currentVolume = SConfig::GetInstance().m_Volume;
+		currentVolume += offset;
+		if (currentVolume > AUDIO_VOLUME_MAX)
+			currentVolume = AUDIO_VOLUME_MAX;
+		UpdateSoundStream();
+	}
+
+	void DecreaseVolume(unsigned short offset)
+	{
+		SConfig::GetInstance().m_IsMuted = false;
```

Comment:
I'd say we do want to unmute. Since there is no feedback on the screen whether you're changing the volume or not it becomes hard to know if you're actually changing it and by how much (though it's possible to open the menu). In addition to that there's no way to know if you're currently muted or not (**no visual indicator anywhere**), I'd say it's easier to handle it if you're always unmuting first. The difference with normal television is that game are almost always able to pause so you could do that first before altering the volume.